standard-bridge-custom-token/scripts/deploy-custom-token.js

```diff
@@ -1,29 +1,187 @@
-async function main() {
-  const l2CustomERC20Factory = await ethers.getContractFactory("L2CustomERC20");
-
-  console.log('deploying L2CustomERC20 to', hre.network.name)
-
-  let l1Token;
-  if (hre.network.name == 'optimism') {
-    l1Token = 'TODO'  // Dev L1 token address
-  } else if (hre.network.name == 'optimistic-kovan') {
-    l1Token = 'TODO'  // Kovan L1 token address
-  } else if (hre.network.name == 'optimistic-mainnet') {
-    l1Token = 'TODO'  // Mainnet L1 token address
-  } else {
-    throw Error("unsupported network")
-  }
+// We require the Hardhat Runtime Environment explicitly here. This is optional
```

Comment:
Why has this example changed?

Reply:
Two reasons:

1. This script is more user friendly, IMAO, because it doesn't need to be customized
2. I wanted the script to also run on a local development node, which means I had to have the code to install an L1 ERC-20 token anyway

Reply:
But the script is now doing lots of other things it is not meant to do, like deploying test tokens, depositing-withdrawing etc.

Reply:
When I learn something and get a code sample, I want to see it working. However, I can put all that in separate functions so it will be easy to just clone the functionality and modify it. Would that be acceptable?